x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/index.tsx

```diff
@@ -91,12 +91,14 @@ export function WaterfallWithSummary({
           </EuiTitle>
         </EuiFlexItem>
         <EuiFlexItem>
-          <EuiPagination
-            pageCount={traceSamples?.length ?? 0}
-            activePage={sampleActivePage}
-            onPageClick={goToSample}
-            compressed
-          />
+          {traceSamples.length > 0 && (
```

Comment:
will `traceSamples` always be defined?
In the previous version we had the optional accessor, if `traceSamples` is undefined we will get a `TypeError`